client/checkout/classic/payment-processing.js

```diff
@@ -203,7 +203,7 @@ async function createStripePaymentElement( api, paymentMethodType ) {
 		amount: amount,
 		paymentMethodCreation: 'manual',
 		paymentMethodTypes: paymentMethodTypes,
-		appearance: initializeAppearance( api ),
```

Comment:
On first page load, when the appearance is not provided from the backend, we now need to wait for the "save" to be completed.